frontend/lib/i18n/locales/nl/election_status.json

```diff
@@ -2,10 +2,18 @@
   "label": "Status",
   "title": "Status verkiezing - Abacus",
   "first_session": "Eerste zitting",
+  "finish_data_entry_first_session": "Steminvoer eerste zitting afronden",
   "definitive": {
     "title": "Alle stembureaus zijn twee keer ingevoerd",
     "message": "De resultaten van alle stembureaus in jouw gemeente zijn correct ingevoerd. Je kunt de uitslag nu definitief maken en het proces-verbaal opmaken. Doe dit alleen als er vandaag niks meer herteld hoeft te worden.",
     "finish_button": "Invoerfase afronden"
   },
-  "finish_first_session_data_entry_status": "<strong>Steminvoer afronden</strong> (eerste zitting)"
+  "coordinator": {
+    "data_entry_in_progress": "Steminvoer bezig",
+    "data_entry_finished": "Steminvoer afgerond"
+  },
+  "typist": {
+    "data_entry_in_progress": "Invoer gestart",
+    "data_entry_finished": "Steminvoer voltooid"
+  }
```

Comment:
We should use the same words for the same status, not different ones depening on role.

Case in point: the translation file uses `"coordinator"`, while the `ElectionStatusWithIcon` wants to know if `isAdministrator`.

Reply:
@jschuurk-kr `isAdministrator` will be removed once we have users with roles, so for now this is just used to show what a coordinator/admistrator person would see. I can make a separate `isCoordinator` flag for those specific views, for now I bundled those under isAdministrator (with the #coordinator or #administrator url flag)

Reply:
@jschuurk-kr For these two statuses I agree, these labels are very similar and could be the same. 

In other cases (not yet implemented) the label for coordinator and typist will differ, because of their different perspective. ie. If an election is fully configured, but the coordinator hasn't opened it for data-entry yet, we will show 'Klaar voor invoer' to the coordinator. To the typist we will communicate 'Wacht op coördinator'. 

So, for the sake of consistency, we could harmonize the labels here, but we should not remove the logic to differentiate the label based on the user's role.

Reply:
Thanks, @jorisleker! I'd still be in favor of removing the logic and add it again when we actually need it. But that's @oliver3's decision, since he took over this PR from Ellen.

Reply:
I've kept the logic but did some other refactor instead.